src/com/google/enterprise/gsafeed/Content.java

```diff
@@ -35,27 +34,59 @@
 public class Content {
 
   @XmlAttribute(name = "encoding")
-  @XmlJavaTypeAdapter(CollapsedStringAdapter.class)
-  protected String encoding;
+  protected Encoding encoding;
   @XmlValue
   protected String value;
 
+  /** Encoding types. */
+  @XmlType(name = "content-encoding")
+  @XmlEnum(String.class)
+  public static enum Encoding {
+    @XmlEnumValue("base64binary")
+    BASE_64_BINARY("base64binary"),
```

Comment:
Base64 seems like one word to me, so I think these (and the one for meta-encoding) should be BASE64.... I also wonder about having a fixed rule for the strings, like the value.replace("-", "_").toUpperCase(US) we talked about, so that this would be BASE64BINARY by virtue of the unhyphenated string value. Not convinced either way.

Reply:
No one else seems to have an opinion, so just s/BASE_64/BASE64/g.